AGENTS.md

```diff
@@ -0,0 +1,238 @@
+# AGENTS.md
+
+This file provides guidance to AI coding agents when working with code in this repository.
+
+## Development Philosophy
+
+We're a startup. We move fast, iterate quickly, and embrace change. When implementing features:
+- Favor pragmatic solutions over perfect architecture
+- Code will likely change - don't over-engineer
+- A/B experiments are common (GrowthBook integration)
+- Test coverage isn't a goal - write tests that validate functionality, not hit metrics
+
+## Project Architecture
+
+This is a pnpm monorepo containing the daily.dev application suite:
+
+| Package | Purpose |
+|---------|---------|
+| `packages/webapp` | Next.js web application (main daily.dev site) |
+| `packages/extension` | Browser extension (Chrome/Opera) built with Webpack |
+| `packages/shared` | Shared React components, hooks, utilities, and design system |
+| `packages/storybook` | Component documentation and development environment |
+| `packages/eslint-config` | Shared ESLint configuration |
+| `packages/eslint-rules` | Custom ESLint rules including color consistency enforcement |
+| `packages/prettier-config` | Shared Prettier configuration |
+
+## Technology Stack
+
+- **Node.js v22.11** (see `.nvmrc`)
+- **pnpm 9.14.4** for package management
+- **TypeScript** across all packages
+- **React 18.3.1** with Next.js 15 for webapp
+- **TanStack Query v5** for server state and data fetching
+- **Jotai** for local state management
+- **GraphQL** with graphql-request for API communication
+- **Tailwind CSS** with custom design system
+- **Jest** for testing
+- **GrowthBook** for feature flags and A/B experiments
+
+## Commonly Used Imports
+
+### Components
+```typescript
+// Buttons (variants: Primary, Secondary, Tertiary, Float, Subtle, Option, Quiz)
+import { Button, ButtonVariant, ButtonSize } from '@dailydotdev/shared/src/components/buttons/Button';
+import { ClickableText } from '@dailydotdev/shared/src/components/buttons/ClickableText';
+
+// Typography
+import { Typography, TypographyType, TypographyColor } from '@dailydotdev/shared/src/components/typography/Typography';
+
+// Form Fields
+import { TextField } from '@dailydotdev/shared/src/components/fields/TextField';
```

Comment:
Note it it can use react-hook-forms and zod validator, example, check `OpportunityEditInfoModal` for example + server side error parsing to form fields

Reply:
applies to other form fields as well